sql/scan.go

```diff
@@ -725,6 +740,32 @@ func (v *qnameVisitor) getDesc(qname *parser.QualifiedName) *TableDescriptor {
 	return nil
 }
 
+// maybeExpandCount provides the special handling for COUNT(*) and
+// COUNT(foo.*), expanding the star into a tuple containing the columns in the
+// referenced table.
+func (v *qnameVisitor) maybeExpandCount(qname *parser.QualifiedName) parser.Expr {
```

Comment:
Yeah, you're correct that it doesn't need to expand to all of the columns. We just need it to expand to a unique set of columns. The primary key would work well.